lib/parser.ts

```diff
@@ -0,0 +1,184 @@
+enum Source {
+  BluRay = 'bluray',
+  HDTV = 'hdtv',
+  WebDL = 'webdl',
+}
+
+interface Result {
+  show: string;
+  episodeCount: number;
```

Comment:
Maybe this should be optional as well, since an `episodeCount` is not meaningful if no `episode` is present?

Reply:
Though I know that it was really added in the previous PR.

Reply:
Would `episodeCount: 0` make sense if there was no episode? 
then it would always be a number for the consumer. Not sure thats better though haha.